test/postgresql.test.js

```diff
@@ -230,6 +230,111 @@ describe('postgresql connector', function() {
     });
   });
 
+  context('pattern matching operators', function() {
+    before(function deleteTestFixtures(done) {
+      Post.destroyAll(done);
+    });
+    before(function createTextFixtures(done) {
+      Post.create([{
+        title: 't1',
+        content: 'T1_TestCase',
+      }, {
+        title: 't2',
+        content: 'T2_TheOtherCase',
+      }], done);
+    });
```

Comment:
You can move the two functions to the bottom of the context block:

```
before(deleteTestFixtures);
before(createTestFixtures);

it('...');
it('...');
...

function deleteTestFixtures(done) {
  Post.destroyAll(done);
}
function create...
```

http://loopback.io//doc/en/contrib/style-guide.html#layout-of-test-files